tests/acceptance/features/webUIRenameFolders/renameFolders.feature

```diff
@@ -58,56 +58,44 @@ Feature: rename folders
     And the user reloads the current page of the webUI
     Then folder "  multiple   spaces    all     over   " should be listed on the webUI
 
-  @skip
-  Scenario: Rename a folder using both double and single quotes
-    When the user renames the following folder using the webUI
-      | from-name-parts | to-name-parts         |
-      | simple-folder   | First 'single' quotes |
-      |                 | -then "double"        |
+  Scenario Outline: Rename a folder using both double and single quotes
+    When the user renames folder <from_name> to <to_name> using the webUI
     And the user reloads the current page of the webUI
-    Then the following folder should be listed on the webUI
-      | name-parts            |
-      | First 'single' quotes |
-      | -then "double"        |
-    When the user renames the following folder using the webUI
-      | from-name-parts       | to-name-parts   |
-      | First 'single' quotes | a normal folder |
-      | -then "double"        |                 |
+    Then folder <to_name> should be listed on the webUI
+    When the user renames folder <to_name> to <next_name> using the webUI
     And the user reloads the current page of the webUI
-    Then folder "a normal folder" should be listed on the webUI
+    Then folder <next_name> should be listed on the webUI
+    Examples:
+      | from_name       | to_name                 | next_name         |
+      | "simple-folder" | "First 'single' quotes" | "a normal folder" |
+      | "simple-folder" | '-then "double"'        |  "abc"            |
 
-  @skip
-  Scenario: Rename a folder using forbidden characters
-    When the user renames folder "simple-folder" to one of these names using the webUI
-      | simple\folder   |
-      | \\simple-folder |
-      | .htaccess       |
-    Then notifications should be displayed on the webUI with the text
-      | Could not rename "simple-folder" |
-      | Could not rename "simple-folder" |
-      | Could not rename "simple-folder" |
+  Scenario Outline: Rename a folder using forbidden characters
+    When the user renames folder <from_name> to <to_name> using the webUI
+    Then notification should be displayed on the webUI with the text <notification_message>
```

Comment:
```Then the error message "Error while renaming "xyz." should be displayed on the webUI```
There's already this line. Check if you can use it. Anyway, an error message is displayed not a notification.

Reply:
Actually error message pops above the input field while renaming folder inside the modal.
But the notification pops at the top of the webUI. So different elements have to be inspected and they're not same.

Reply:
But yes, the language can be modified,,,,notification can be replaced with error message! What do you think?

Reply:
:radio: